pkg/kinds/playlist/types.deepcopy.go

```diff
@@ -0,0 +1,77 @@
+package playlist
+
+import (
+	"encoding/json"
+	"fmt"
+
+	runtime "k8s.io/apimachinery/pkg/runtime"
+)
+
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
+func (in *Playlist) DeepCopyInto(out *Playlist) {
+	b, err := json.Marshal(in)
```

Comment:
@toddtreece -- the update is broken since my hand crafted one does not work as expected :(

is it hard to regenerate?